### DISCUSSION TITLE
Big Icons Update

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#145] [General]
Icons now have, apart from the icon and the side, a show setting:

![icons234](https://github.com/user-attachments/assets/0ce3be4b-77e9-433b-833a-a9feecb7d2b6)

This controls when the icon is visible.

You can now, for instance, only show a certain icon on an item if they're selected, or hovered, or both (focus).

It defaults to `Always`.

Also added more click actions for lots of icons, all icons do something when clicked.

**Comment by N3C2L** [comment]
Thanks! These options should satisfy everyone's icons preferences!

**Comment by madprops** [comment]
Yeah I'm using `focus` on node icon, instead of always displaying it, it's cleaner.
